fem/gridfunc.cpp

```diff
@@ -324,45 +324,12 @@ void GridFunction::ComputeFlux(BilinearFormIntegrator &blfi,
 
 int GridFunction::VectorDim() const
```

Comment:
Can you add Doxygen comments for this and for `GridFunction::CurlDim` (something like "Returns FiniteElementSpace::GetVectorDim()" would be fine).

Reply:
Done in #4598